web/templates/stream_settings/channel_name_conflict_error.hbs

```diff
@@ -0,0 +1,22 @@
+{{#if is_archived}}
+    {{#if can_view_channel}}
+        {{#tr}}
+            An <z-link>archived channel</z-link> with this name already exists.
+            {{#*inline "z-link"}}<a href="#channels/{{stream_id}}/general" class="stream-settings-link">{{> @partial-block}}</a>{{/inline}}
+        {{/tr}}
+    {{else}}
+        {{t "An archived channel with this name already exists." }}
+    {{/if}}
+    {{#if show_rename}}
+        <span id="archived_stream_rename" data-stream-id="{{stream_id}}">{{t "Rename it" }}</span>
```

Comment:
Since we style this as link using CSS, can we directly use `a` tag here instead of `span` so we do need to add some of these CSS? Previously we used `a` tag.

Also, there is existing `a#archived_stream_rename` element which can be removed now since we do not use it.